streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1076,7 +1082,12 @@ public class StreamsConfig extends AbstractConfig {
                     Type.LONG,
                     null,
                     Importance.LOW,
-                    WINDOW_SIZE_MS_DOC);
+                    WINDOW_SIZE_MS_DOC)
+            .define(DEFAULT_CLIENT_SUPPLIER_CONFIG,
```

Comment:
nit: we keep the configs grouped by priority and order alphabetically within each group. Can we more this accordingly?

Reply:
Sure